src/js/footer/osSignupRedirections/index.js

```diff
@@ -0,0 +1,44 @@
+import { getCookieByKey } from "../cookies";
+
+const allowedLangs = ["en", "fr", "de", "it", "es", "pt", "pl", "ru"];
+
+document.addEventListener("DOMContentLoaded", function () {
+  let languageCookie = getCookieByKey(document.cookie, "webflow-user-language");
+  let signupLink = "";
+  if (
+    languageCookie.toLowerCase() === "en" &&
+    window.location.pathname.includes("/eu/")
+  ) {
+    signupLink = window.location.origin + "/eu/signup";
+  } else {
+    if (languageCookie.toLowerCase() === "en") {
+      signupLink = window.location.origin + "/signup";
+    } else {
+      if (window.location.pathname.includes("/eu/")) {
+        signupLink =
+          window.location.origin + "/" + languageCookie + "/eu/signup";
+      } else {
+        signupLink = window.location.origin + "/" + languageCookie + "/signup";
```

Comment:
Hi @moiz-deriv , why is language cookie i the middle?

Reply:
this is for redirection to deriv.com signup for unsupported langs